src/components/ResultSegment-review.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const ResultSegment = ({ type, homeValue, homeMax, awayValue, awayMax }) => {
+  return (
+    <div className="segment-display">
+      <div>{type}</div>
+
+      <div className="bar-display">
+        <div>
+          <div>{type === "Possession" ? homeValue + "%" : homeValue}</div>
+          <progress
+            className="rtl-progressbar"
+            value={homeValue}
+            max={homeMax}
+          />
+        </div>
+        <div>
+          <div>{type === "Possession" ? awayValue + "%" : awayValue}</div>
+          <progress className="colored" value={awayValue} max={awayMax} />
+        </div>
```

Comment:
You could even extract this into it's own component, both divs are doing the same thing - one is showing the home data, one is showing the away data. could you do something like this?
```
        <StatsBreakdown type={type} value={homeValue} total={total} />
        <StatsBreakdown type={type} value={awayValue} total={total} />
```